charts/operator-wandb/charts/weave/templates/deployment.yaml

```diff
@@ -60,7 +60,7 @@ spec:
             - name: WEAVE_LOCAL_ARTIFACT_DIR
               value: /vol/weave/cache
             - name: WEAVE_AUTH_GRAPHQL_URL
-              value: {{ .Values.global.host }}/graphql
+              value: http://{{ include "weave.appFullname" . }}.default.svc.cluster.local:8080/graphql
```

Comment:
The `default.svc.cluster.local` piece should be parameterized, at the very least the `default` part.  `.Release.Namespace` will probably give you what you want.